### ISSUE TITLE
Accessibility issue with kirby-checkbox - Aria-checked used where it is not allowed

### ISSUE THREAD

**Issue by fr1m3r** [kirbydesign/designsystem#3844]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
The Level Access tool is reporting our use of kirby-checkbox on several of our consents in DRB, regarding the use of aria-checked where it is not allowed.
It seems that it is not possible to solve it individually, where kirby-checkbox is used. Therefore this issue.

## Describe how to reproduce the bug
1. Login to DRB Web (no matter which bank)
2. Go to 'Profile' in the left menu
3. Go to 'Consent and agreements'
4. Go to the consent called 'Marketing consent'
5. Scan the 'Marketing consent' page with the Level Access tool (extension to Chrome)
6. See the reported issues in the Level Access tool

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
That aria-checked should have been handled correctly on its own.

## Add any screenshots

![Image](https://github.com/user-attachments/assets/905f8b0c-6694-44ab-aa37-85a279e9aac3)
![Image](https://github.com/user-attachments/assets/925025da-3a56-4492-9a48-f76cb48bc1e6)

## Please complete the following information:
- OS: [Windows 11]
- Browser [Chrome]
- Browser version [133.0.6943.127]

## Are there any additional context?
No

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by Loujuu** [comment]
I can not reproduce the bug. I have testet with Level Access tool and no bug related to the issue is current. Task is closed.
